### PR TITLE
[fluentd-splunk-objects] Add, [fluentd-splunk-logs] Upgrade to 1.1.0

### PR DESCRIPTION
**Pull request by Nuru**
## what
- Add support for logging Kubernetes objects to Splunk
- Upgrade fluentd-splunk-logs to use current 1.1.0 version of Splunk conector

## why
Customer request

### CODE REVIEW

File: releases/fluentd-splunk-objects.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Fluentd to Splunk logs                                                            ##
+## Forward logs to Splunk with fluentd                                               ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/splunk/splunk-connect-for-kubernetes/blob/master/helm-chart/splunk-kubernetes-logging/values.yaml
+#
+
+- name: "fluentd-splunk-objects"
```
Comment (osterman) [inline releases/fluentd-splunk-objects.yaml:13]: why not just call it `fluentd-splunk` to keep it short & sweet?

Comment (Nuru) [reply releases/fluentd-splunk-objects.yaml:13]: Need to differentiate it from fluentd-splunk-logs

Comment (osterman) [reply releases/fluentd-splunk-objects.yaml:13]: oh, sounds like it has nothing to do with `fluentd` any more?

Comment (Nuru) [reply releases/fluentd-splunk-objects.yaml:13]: Splunk [uses fluentd](https://github.com/splunk/splunk-connect-for-kubernetes/tree/master/helm-chart/splunk-kubernetes-objects#splunk-collect-for-kubernetes-objects)

> The deployment runs fluentd with the kubernetes objects input plugin and the Splunk HEC output plugin to collect objects and send them via Splunk HEC.

File: releases/fluentd-splunk-objects.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    chart: "splunk-kubernetes-objects"
+    repo: "splunk"
+    component: "splunk"
+    namespace: "kube-system"
+    default: "false"
+  chart: "https://github.com/splunk/splunk-connect-for-kubernetes/releases/download/1.1.0/splunk-kubernetes-objects-1.1.0.tgz"
+  wait: true
+  installed: {{ env "FLUENTD_SPLUNK_INSTALLED" | default "true" }}
+  values:
+  - fullnameOverride: "splunk-k8s-objects"
```
Comment (osterman) [inline releases/fluentd-splunk-objects.yaml]:
```suggestion
  - fullnameOverride: "fluentd-splunk-objects"
```

I think it's more confusing to introduce yet another variation that is not any shorter and is not needed for deterministic service discovery

Comment (Nuru) [reply releases/fluentd-splunk-objects.yaml]: I updated the names so that the services etc. have the names Splunk support would be expecting.

**Review by osterman — APPROVED** [review]

**Pull request merged**